### ISSUE TITLE
Flutter web compatibility

### ISSUE THREAD

**Issue by bean5** [Daegalus/dart-uuid#42]
**Expected**
Builds and runs for web. README says "Runs in web, server, and flutter".

**Actual**
Crashes upon build. 

_System_
```
Flutter 1.14.6 • channel beta • https://github.com/flutter/flutter.git
Framework • revision fabeb2a16f (3 weeks ago) • 2020-01-28 07:56:51 -0800
Engine • revision c4229bfbba
Tools • Dart 2.8.0 (build 2.8.0-dev.5.0 fc3af737c7)
```

_Error_
```
Unable to find modules for some sources, this is usually the result of either a
bad import, a missing dependency in a package (or possibly a dev_dependency
needs to move to a real dependency), or a build failure (if importing a
generated file).

Please check the following imports:

`import 'package:uuid/uuid_util.dart';` from aWebProject|main.dart at
9:1
`import 'package:uuid/uuid.dart';` from aWebProject|main.dart at 8:1


Failed after 294ms                                                      
Building application for the web...                                 9.4s
Failed to build application for the Web.
```

**Comment by bean5** [comment]
Other than this, the project looks pretty solid!

**Comment by bean5** [comment]
Advised: Add test for web.

**Comment by Daegalus** [comment]
I'll take a look soon, especially since I'm getting into flutter myself. The library should work no problems on web or flutter.

Also need to dig into web/flutter testing and see what I need to add to my current tests.

Thanks for the report. Hopefully I can get something out as a fix real soon if I can reproduce it

**Comment by Daegalus** [comment]
I cant seem to reproduce it. I installed Flutter Beta, same version as yours. I then created a project. Added uuid to the package.json at 2.0.4, then added the import, and a call to the function in a string.

```yaml
uuid: ^2.0.4
```dart
import 'package:uuid/uuid.dart';

...

Text(
              'You have pushed the button this many times: '  + (new Uuid()).v4(),
            ),
```

[![Flutter Web UUID](https://i.postimg.cc/brX878gv/Screenshot-1582087511.png)](https://postimg.cc/5YmZzhZc)

**Comment by bean5** [comment]
Hmm. Last I checked it is working for me, as well. I ran into a strange problem yesterday whose work-around was to rename my project's folder. Before rename, web builds were failing. After rename, web builds worked. I think it has something to do with the `.pub-cache/` folder Flutter+dart manage behind the scenes.

So, your project is probably fine. Sorry I opened in error. Hopefully flutter will get better at caching project builds in the future.
